boilerplate/openshift/golang-osd-operator-osde2e/e2e-harness-generate.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+set -e
+
+cmd=${0##*/}
+
+usage() {
+    cat <<EOF
+Usage: $0 OPERATOR_NAME OSDE2E_CONVENTION_DIR
+ 
+EOF
+    exit -1
+}
+
+OPERATOR_NAME=$1
+OSDE2E_CONVENTION_DIR=$2
+REPO_ROOT=$(git rev-parse --show-toplevel)
+HARNESS_DIR=$REPO_ROOT/osde2e
+
+# Update operator name in templates
+export OPERATOR_HYPHEN_NAME=$(echo "$OPERATOR_NAME"| sed 's/-/_/g')
+export OPERATOR_PROPER_NAME=$(echo "$OPERATOR_NAME"| sed 's/-/ /g' |awk '{for(i=1;i<=NF;i++){ $i=toupper(substr($i,1,1)) substr($i,2) }}1')
+export REPLACE_SPECNAME=${OPERATOR_PROPER_NAME} Test Harness
+export REPLACE_FUNC=$(echo "$REPLACE_SPECNAME" | sed 's/ //g' )
+
+mkdir $HARNESS_DIR
+mkdir $HARNESS_DIR/tests
+ 
+
+echo "
+# THIS FILE IS GENERATED BY BOILERPLATE. DO NOT EDIT.
+FROM registry.ci.openshift.org/openshift/release:golang-1.18 AS builder
+    
+ENV PKG=/go/src/github.com/openshift/${OPERATOR_NAME}/
+WORKDIR \${PKG}\
+    
+COPY . .
+    
+FROM registry.access.redhat.com/ubi7/ubi-minimal:latest
+    
+COPY ./harness.test harness.test
+    
+ENTRYPOINT [ \"/harness.test\" ]" > $HARNESS_DIR/Dockerfile
```

Comment:
```suggestion
# THIS FILE IS GENERATED BY BOILERPLATE. DO NOT EDIT.
FROM registry.ci.openshift.org/openshift/release:golang-1.18 AS builder

ENV PKG=/go/src/github.com/openshift/${OPERATOR_NAME}/
WORKDIR \${PKG}\

COPY . .

FROM registry.access.redhat.com/ubi7/ubi-minimal:latest

COPY ./harness.test harness.test

ENTRYPOINT [ \"/harness.test\" ]" > $HARNESS_DIR/Dockerfile
```

Reply:
Hey @jbpratt , thanks for the suggestion. We can't make changes to boilerplate here, since this repo is consuming the central boilerplate. But the convention is still new, so you can still make the suggestion to the main repo here https://github.com/openshift/boilerplate